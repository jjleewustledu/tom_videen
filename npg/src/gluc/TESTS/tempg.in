p5027.tsc
p5027g.dta
p5027.pbl
p5027A.log
p5027B.log
p5027C.log
p5027D.log
