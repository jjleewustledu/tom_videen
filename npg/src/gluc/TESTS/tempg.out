 
  ENTER NAME OF FILE CONTAINING BLOOD CURVE DATA
  FILE 
 p5027g.dta                                                                      
   HAS BEEN OPENED, HEADER IS: 
  @01@ p5027 11-07-95 corrier cg1 961 jth                               
 
  ENTER NAME OF FILE WITH PARAMETER BLOCK INFO
  RETURN FROM 1ST PBLK CALL
  ENTER DESTINATION FOR PRIMARY PRINT-OUT
  ENTER DESTINATION FOR SECONDARY PRINT-OUT
  RETURN FROM PBLK CALL
  RETURN FROM MARQ 
     0.10000E+01    -0.99989E+00     0.99901E+00
    -0.99989E+00     0.10000E+01    -0.99954E+00
     0.99901E+00    -0.99954E+00     0.10000E+01
    -0.86078E+00     0.86351E+00    -0.86995E+00
  SINGULAR VALUES     3.8001258356772   0.19901620087965    8.4879784552039D-04
     9.1655976100872D-06
  condition no =     414608.
  DETERMINANT OF FI MATRIX     5.8837127906067D-09
